src/page/dataverse/dataspace/dataspace.tsx

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useState } from 'react'
+import { useParams } from 'react-router-dom'
+import { getResourceDetails } from '../dataverse'
+import type { DataverseItemDetails } from '../dataverse'
+
+const Dataspace = (): JSX.Element => {
+  const { id } = useParams<string>()
+  const [dataspace, setDataspace] = useState<DataverseItemDetails | undefined>(undefined)
```

Comment:
Use implicit argument.

```suggestion
  const [dataspace, setDataspace] = useState<DataverseItemDetails | undefined>()
```